cmd/ops/unused.go

```diff
@@ -95,8 +95,11 @@ func unusedMarkdownOut(out output.Data) {
 
 	out.Walk(func(schema string, f output.Field, fieldIdx int) {
 		if fieldIdx == 0 {
+			if hasUnused {
```

Comment:
Just to make sure I understand this. You want to add a line if this was set to true before it is set to true right after?

Reply:
Yes. This should add a new line before each schema except for the first schema. This is what we want to achieve:

```
Schema: some schema
   - field bla bla
   - field bla bla

# <- Blank line here
Schema: some other schema
   - field bla bla
   - field bla bla

# <- Blank line here
Schema: another schema
   - field bla bla
   - field bla bla
```

Reply:
Let me add a comment since that wasn't obvious 😅

Reply:
I tweaked the comment to hopefully make it more obvious.